internal/driver/mobile/app/darwin_ios.go

```diff
@@ -224,14 +211,15 @@ func drawloop() {
 	runtime.LockOSThread()
 	defer runtime.UnlockOSThread()
 
-	for workAvailable := theApp.worker.WorkAvailable(); ; {
+	workAvailable := theApp.worker.WorkAvailable()
+	for {
 		select {
 		case <-workAvailable:
 			theApp.worker.DoWork()
 		case <-theApp.publish:
 			theApp.publishResult <- PublishResult{}
 			return
-		case <-time.After(100 * time.Millisecond): // incase the method blocked!!
+		case <-time.After(time.Millisecond): // incase the method blocked!!
```

Comment:
Why moving to wait only a millisecond? Won't this mean we drop paint frames as they often take >= 10ms

Reply:
On iOS it is less responsive when interacting with the UI. Mainly because of this block. If there are no draw events, this function returns after 100 ms and causes laggy experiences.

Reducing the number here does not affect drawing events if `workAvailable` is selected and starts execution, then the function will only return if `worker.DoWork()` is returned.

Reply:
Has this been verified on old iOS devices? - foreground and background?
The change it was added was titled "Fix background mobile hangs"...

Reply:
Yes, the previous change was about restoring the correct OpenGL context. It is quite mysterious why there is a 100 ms for a timeout. I think the block should be restored to 1 ms approx. equal to 1s/960, in this way, the user input events can be handled timely.

Reply:
But sadly I don't have an old iOS device...